spec/spec_helper.rb

```diff
@@ -2,15 +2,15 @@
 
 require 'simplecov'
 
-require File.expand_path("../dummy/config/environment.rb", __FILE__)
+require File.expand_path('../dummy/config/environment', __FILE__)
 require 'rspec/rails'
 
 Dir.glob(File.expand_path('support/**/*.rb', __dir__), &method(:require))
 
-RSpec.configure do |config|
-  # redirect console output for unit tests
-  # config.before(:all, type: :unit, &:silence_output)
+ActiveRecord::Schema.verbose = false
```

Comment:
Why do we need this and next lines?

Reply:
`ActiveRecord::Schema.verbose = false' this line does not show puts in tests about migration. Without next line lol_dba test say that does not see table that he need.